stable/node-problem-detector/README.md

```diff
@@ -1,42 +1,16 @@
 # node-problem-detector
 
-![Version: 2.3.12](https://img.shields.io/badge/Version-2.3.12-informational?style=flat-square) ![AppVersion: v0.8.15](https://img.shields.io/badge/AppVersion-v0.8.15-informational?style=flat-square)
+![Version: 2.3.13](https://img.shields.io/badge/Version-2.3.13-informational?style=flat-square) ![AppVersion: v0.8.17](https://img.shields.io/badge/AppVersion-v0.8.17-informational?style=flat-square)
 
 This chart installs a [node-problem-detector](https://github.com/kubernetes/node-problem-detector) daemonset. This tool aims to make various node problems visible to the upstream layers in cluster management stack. It is a daemon which runs on each node, detects node problems and reports them to apiserver.
 
 **Homepage:** <https://github.com/kubernetes/node-problem-detector>
 
-## How to install this chart
```

Comment:
Thanks for the PR @stxm but could you work out how to not have helm-docs remove this part?